bigframes/ml/compose.py

```diff
@@ -46,6 +46,28 @@
 )
 
 
+class SQLScalarColumnTransformer(base.BaseTransformer):
```

Comment:
If inherits from BaseTransformer, then to_gbq() method is provided. While I don't think this class supports the operation as it is only one column SQL.